### PR TITLE
Upgrade to 0.5.8

### PR DESCRIPTION
**Pull request by haritamar**

### CODE REVIEW

File: pydruid/utils/filters.py [hunk 1]
```diff
@@ -75,17 +75,10 @@ def __init__(self, extraction_function=None, **args):
...
-                    "lower": args["lower"],
-                    "lowerStrict": args["lowerStrict"],
-                    "upper": args["upper"],
-                    "upperStrict": args["upperStrict"],
-                    "alphaNumeric": args["alphaNumeric"],
-                    "ordering": args["ordering"],
-                }
-            )
+            self.filter["filter"]["dimension"] = args["dimension"]
+            for key in ["lower", "lowerStrict", "upper", "upperStrict", "ordering"]:
```
Comment (bikov) [inline pydruid/utils/filters.py]: why `alphaNumeric` is missing?

Comment (bikov) [reply pydruid/utils/filters.py]: Oh I see it in the next commit

File: pydruid/client.py [hunk 2]
```diff
@@ -420,17 +581,68 @@ def _post(self, query):
...
-            _, e, _ = sys.exc_info()
-            err = None
+        except urllib.error.HTTPError as e:
+            err = e.read()
             if e.code == 500:
                 # has Druid returned an error?
                 try:
-                    err = json.loads(e.read().decode("utf-8"))
+                    err = json.loads(err)
+                except ValueError:
```
Comment (keren-meron) [inline pydruid/client.py:594]: no need to catch ValueError again in line 593

File: pydruid/client.py [hunk 3]
```diff
@@ -420,17 +581,68 @@ def _post(self, query):
...
-        except urllib.error.HTTPError:
-            _, e, _ = sys.exc_info()
-            err = None
+        except urllib.error.HTTPError as e:
+            err = e.read()
             if e.code == 500:
                 # has Druid returned an error?
                 try:
-                    err = json.loads(e.read().decode("utf-8"))
+                    err = json.loads(err)
```
Comment (keren-meron) [inline pydruid/client.py:593]: not sure why we don't decode? anyway I see that it's already been that way so fine

**Review by keren-meron — APPROVED** [review]

File: pydruid/client.py (hunk 3 above)
Comment (haritamar) [reply pydruid/client.py:593]: hmm not sure either
I think it was part of the merge
will have a look

File: pydruid/client.py (hunk 2 above)
Comment (haritamar) [reply pydruid/client.py:594]: thanks
looks like it's also like that in pydruid's master branch, weird
anyway I'll fix

**Pull request merged**